controls/radheatmap/tooltip.md

```diff
@@ -0,0 +1,148 @@
+---
+title: ToolTip
+page_title: ToolTip
+description: ToolTip
+slug: radheatmap-tooltip
+tags: tooltip, heatmap, tooltipcell, celltooltip, heatmaptooltipsettings
+published: True
+position: 5
+---
+
+# ToolTip
+
+The RadHeatMap control provides a tooltip behavior which can be used to visualize arbitrary information related to a data point. The ToolTip functionality is disable by default. To enable it, you need to set __ShowToolTips__ property of the RadHeatMap to true. 
+
+#### __[XAML] Example 1: Enabling the ToolTip behavior of the RadHeatMap__
+
+{{region radheatmap-tooltip_0}}
+	 <telerik:RadHeatMap ShowToolTips="True" />
+{{endregion}}
+
+## Customizing the Tooltip
+
+To customize the apperance of the tooltip you can use the __HeatMapToolTipSettings__ property. For demonstration purposes we will use a simple class that will represent one cell of the RadHeatMap control. This class will expose three properties:
+
+ * __Row:__ А string property which gets or sets the row name of the corresponding cell.
+ * __Column:__ А string property which gets or sets the column name of the corresponding cell.
+ * __Value:__ Аn integer property which gets or sets the value of the corresponding cell.
+
+#### __[C#] Example 2: Creating the model__
+{{region radheatmap-tooltip-1}}
+	public class TempInfo
+    {
+        public int Year { get; set; }
+        public string Month { get; set; }
+        public double Temperature { get; set; }
+    }
+{{endregion}}
+
+#### __[VB.NET] Example 2: Creating the model__
+{{region radheatmap-tooltip-2}}
+    Public Class TempInfo
+		Public Property Year As Integer
+		Public Property Month As String
+		Public Property Temperature As Double
+	End Class
+{{endregion}}
+
+To fill the RadHeatMap with sample data we can create a list of *PlotInfo* objects and set it as DataContext of the control. 
+
+#### __[C#] Example 3: Populating the RadHeatMap control__
+{{region radheatmap-tooltip-3}}
+	public partial class MainWindow : Window
+    {
+        public MainWindow()
+        {
+            this.InitializeComponent(); 
+			this.PrepareData();
+        }
+		
+		private void PrepareData()
+		{
+			int year = 2018;
+			string[] months = new string[6] { "Jan", "Feb", "Mar", "Apr", "May", "Jun" };
+			var randomNumberGenerator = new Random();
+
+			var source = new ObservableCollection<TempInfo>();
+			for (int i = 0; i < months.Length; i++)
+			{
+				for (int k = 0; k < 6; k++)
+				{
+					var info = new TempInfo() { Year = year + k, Month = months[i], Temperature = randomNumberGenerator.Next(10, 300) };
+					source.Add(info);
+				}
+			}
+
+			this.categoricalDefinition.ItemsSource = source;
```

Comment:
What is "categoricalDefinition"? Add some information or a code snippet about this.

Reply:
Now I saw the the "categoricalDefinition" is defined in the next code snippet. Maybe, you can switch the order. Add the xaml snippet before the population with the data.